test/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 import test from 'tape';
-import hello from '../index';
+import hello from '../source/index';
```

Comment:
Because I don't want to build tests (to capture original content for coverage), import path should be updated to the correct one.